tests/common/import_hook.rs

```diff
@@ -0,0 +1,134 @@
+use crate::common::{create_virtualenv, test_python_path};
+use anyhow::{bail, Result};
+use maturin::{BuildOptions, CargoOptions, Target};
+use serde_json;
+use serde_json::{json, Value};
+use std::collections::BTreeMap;
+use std::path::{Path, PathBuf};
+use std::process::Command;
+use std::{env, fs, str};
+
+pub fn test_import_hook(
+    virtualenv_name: &str,
+    test_script_path: &Path,
+    extra_packages: Vec<&str>,
+    extra_envs: BTreeMap<&str, &str>,
+    verbose: bool,
+) -> Result<()> {
+    let python = test_python_path().map(PathBuf::from).unwrap_or_else(|| {
+        let target = Target::from_target_triple(None).unwrap();
+        target.get_python()
+    });
+
+    let (venv_dir, python) = create_virtualenv(virtualenv_name, Some(python)).unwrap();
+
+    println!("installing maturin binary into virtualenv");
```

Comment:
I'm not sure if this is the best way of making the `maturin` binary available to the python tests on the CI but currently they fail without this.
I tried installing with `pip install .` but that seems to install a release binary from pypi or something because `maturin new` was not available